packages/vue/README.md

```diff
@@ -1,3 +1,6 @@
+![](https://img.shields.io/npm/dw/@aws-amplify/ui-vue?label=Download&logo=Amplify)  
+![](https://img.shields.io/npm/v/@aws-amplify/ui-vue/latest?style=flat)
```

Comment:
These show on two lines for me when I look at the rich diff (assuming you mean for them to be single line like the others)

Reply:
These should render in a single line in markdown, there would have to be another new line in between these two for them to render separately.

Reply:
Hmm maybe just an issue with the preview then? Seems to be just for this one. 
<img width="979" alt="Screen Shot 2022-10-14 at 11 39 07 AM" src="https://user-images.githubusercontent.com/376920/195886422-971ff4db-ba03-4e1d-bdb1-77ac078b40fa.png">

Reply:
Those are the same format as in the PR description, so should be in a table

Reply:
Interesting, looks like the README can be rendered to the same line since that's how we do it in the current README [code](https://github.com/aws-amplify/amplify-ui/blob/194ba9fff4ea20fd87d6b40d4c98b19f77bad727/README.md?plain=1#L7-L11).

But to be safe, I can change the code to guarantee that